vispy/geometry/generation.py

```diff
@@ -89,6 +89,250 @@ def create_cube():
     return vertices, filled, outline
 
 
+def create_plane(width=1,
+                 height=1,
+                 width_segments=1,
+                 height_segments=1,
+                 direction='+z'):
+    """ Generate vertices & indices for a filled and outlined plane.
+
+    Parameters
+    ----------
+    width : numeric
+        Plane width.
+    height : numeric
+        Plane height.
+    width_segments : int
+        Plane segments count along the width.
+    height_segments : numeric
+        Plane segments count along the height.
+    direction: unicode
+        ``{'-x', '+x', '-y', '+y', '-z', '+z'}``
+        Direction the plane will be facing.
+
+    Returns
+    -------
+    vertices : array
+        Array of vertices suitable for use as a VertexBuffer.
+    faces : array
+        Indices to use to produce a filled plane.
+    outline : array
+        Indices to use to produce an outline of the plane.
+
+    References
+    ----------
+    [1] Cabello, R. (n.d.). PlaneBufferGeometry.js. Retrieved May 12, 2015,
+        from https://github.com/mrdoob/three.js/blob/master/src/extras/geometries/PlaneBufferGeometry.js  #noqa
```

Comment:
I need to double check but I think it is fine this way, just on the offending line.